theoplayer/getting-started/01-sdks/03-ios/00-getting-started.mdx

```diff
@@ -0,0 +1,196 @@
+# Getting started on iOS/tvOS
+
+## Introduction
+
+This guide covers the steps to obtain and setup the core THEOplayer SDK targeting a native iOS/tvOS application. It also provides information and usage of the commonly used APIs.
+
+## Setup
+
+### THEOplayer license
+
+THEOplayer requires a valid THEOplayer license to be configured in order to access the SDK APIs (with the exception of some whitelisted sources hosted by our CDN). Request your license via [https://portal.theoplayer.com](https://portal.theoplayer.com).
+
+After creating your license, you can copy its license string to your clipboard as demonstrated in the screenshot below.
+
+![](https://cdn.theoplayer.com/images/git/theoplayer-ios-sdk-license-string.png)
+
+> **Note:** It's possible to add multiple platforms in a single license ie. iOS & tvOS.
+
+Later in the guide we explain how to configure the license ([in this section](#adding-the-theoplayer-license)).
+
+### Installation
+
+The THEOplayer SDK is published on the following package managers:
+
+#### <span style={{ fontSize: '1.25rem' }}>Cocoapods</span>
+
+> **_It's considered good practice to run `pod repo update` before adding new pods to your Podfile.
+This will help find newly added pods by updating the list of references to the public spec repos.
+If you just installed Cocoapods or are already up-to-date, then this step can be skipped._**
```

Comment:
```suggestion
:::tip

It's considered good practice to run `pod repo update` before adding new pods to your Podfile.
This will help find newly added pods by updating the list of references to the public spec repos.
If you just installed Cocoapods or are already up-to-date, then this step can be skipped.

:::
```